contracts/ClaimZap.sol

```diff
@@ -0,0 +1,219 @@
+// SPDX-License-Identifier: MIT
+pragma solidity 0.8.7;
+
+import "./helper/MathUtil.sol";
+import '@openzeppelin/contracts/token/ERC20/IERC20.sol';
+import '@openzeppelin/contracts/token/ERC20/utils/SafeERC20.sol';
+import '@openzeppelin/contracts/utils/math/SafeMath.sol';
+
+
+interface IBasicRewards{
+    function getReward(address _account, bool _claimExtras) external;
+    function getReward(address _account) external;
+    function getReward(address _account, address _token) external;
+    function stakeFor(address, uint256) external;
+}
+
+interface IVeTokenRewards {
+    function getReward(address _account, bool _claimExtras, bool _stake) external;
+}
+
+// ToDo: delete when we confirm it's obsolete
+//interface IChefRewards{
+//    function claim(uint256 _pid, address _account) external;
+//}
+
+interface IVe3TokenDeposit {
+    function deposit(uint256, bool) external;
+}
+
+interface ISwapExchange {
+
+    function exchange(
+        int128,
+        int128,
+        uint256,
+        uint256
+    ) external returns (uint256);
+}
+
+interface IVE3DLocker {
+    function getReward(address _account, bool _stake) external;
+    function lock(address _account, uint256 _amount) external;
+}
+
```

Comment:
please move the interfaces to interfaces folder and you may find same interfaces there